src/modules/search/components/ClientSearch.tsx

```diff
@@ -106,22 +95,23 @@ export const CLIENT_COLUMNS: {
     ),
     key: 'dob',
     render: (client) => <ContextualClientDobAge client={asClient(client)} />,
+    // Don't link cell even if row is linked, because of internal click target
     dontLink: true,
+    // Fixed width so it doesn't move around when visibility is toggled
+    width: '180px',
   },
 };
 
-export const SEARCH_RESULT_COLUMNS: ColumnDef<ClientFieldsFragment>[] = [
-  CLIENT_COLUMNS.id,
-  CLIENT_COLUMNS.first,
-  CLIENT_COLUMNS.last,
-  { ...CLIENT_COLUMNS.dobAge, width: '180px' },
-];
+export const SEARCH_RESULT_COLUMNS: ColumnDef<ClientSearchResultFieldsFragment>[] =
+  [
+    CLIENT_COLUMNS.id,
+    CLIENT_COLUMNS.first,
```

Comment:
Per our [tables sheet](https://docs.google.com/spreadsheets/d/17ZpdB0fH6kUL14KDRWrlukwIH93BNx9grrNZApLpUwg/edit?gid=1987637083#gid=1987637083), I think we had been planning to consolidate around consistently using one Client Name col, instead of sometimes having first/last be separate cols. Do you want to make that change here? (Especially since that's already how the mobile columns are configured.) 

I could also see the justification for keeping first/last separate on desktop search to support some workflows; maybe add a comment about why, if you want to keep it as-is.

Reply:
Yes, I got cold feet about that [slack msg](https://greenriver.slack.com/archives/C033YNSN0Q1/p1738589307247659?thread_ts=1737558629.575329&cid=C033YNSN0Q1). I reverted to reduce the impact of the change, thinking maybe we should follow up on how often people search by DOB. We didn't talk about the fact that you can search by DOB/ID but they never show up in the results anymore.

Reply:
Though replacing this with `[CLIENT_COLUMNS.name, CLIENT_COLUMNS.id, CLIENT_COLUMNS.dobAge]` seems fine to me, it's showing the same information. I will make that change. It is the full removal of DOB and ID that had me slightly concerned about disrupting workflows